sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -463,7 +463,7 @@ otp_json(
             name = "CREATOR_SW_CFG",
             items = {
                 "CREATOR_SW_CFG_RMA_SPIN_EN": hex(CONST.TRUE),
-                "CREATOR_SW_CFG_RMA_SPIN_CYCLES": "10",
+                "CREATOR_SW_CFG_RMA_SPIN_CYCLES": "0x2000000",
```

Comment:
this only increases the value for this particular test, the `CREATOR_SW_CFG` values provisioned during manufacturing may be different, so this should be safe. Though this may act as a good reference point. @alphan can confirm tho